proposals/materialx-versioning/README.md

```diff
@@ -0,0 +1,126 @@
+# MaterialX Versioning in OpenUSD
+
+## Contents
+- [TL;DR](#tldr)
+- [Introduction](#introduction)
+- [Problem](#problem)
+- [Proposal to Version MaterialX data in OpenUSD](#proposal-to-version-materialx-data-in-openusd)
+    - [Overview](#overview) 
+- [Risks](#risks)
+    - [Risk 1](#risk_1)
+- [Out of Scope](#out-of-scope)
+- [Questions](#questions)
+
+## TL;DR
+We propose adding attributes to UsdShade `Material` primitives to record the MaterialX library version they were authored
+with. This will allow for MaterialX to provide an automatic upgrade functionality to the data stored in OpenUSD.
+
+The following is an example of a `Material` primitive with the applied schema applied.
+```
+def "Material" MyMaterial ( 
+    prepend apiSchemas = ["MaterialXVersionAPI"]
```

Comment:
Just spitballing, but could it be useful to have this be `MaterialXInfoAPI` so that in the future we could add more data hints as needed? One thing that comes to mind is the idea of being able to point to specific MaterialX shader libraries, or perhaps if MaterialX adds any other parameters that apply globally.